jobbergate-api/jobbergate_api/tests/apps/job_scripts/test_routers.py

```diff
@@ -143,7 +142,7 @@ async def test_create_job_script(
     assert job_script.job_script_owner_email == "owner1@org.com"
     assert job_script.job_script_description is None
     assert job_script.job_script_data_as_string
-    assert job_script.job_script_data_as_string != job_script_data["job_script_data_as_string"]
+    assert job_script.job_script_data_as_string.startswith('{"application.sh":')
```

Comment:
Yeah, very good point, no need for it.

Reply:
Would `assert isinstance(job_script.job_script_data_as_string, str)` be better? Because the original comparison is not possible anymore, `job_script_data` does not contain the key `"job_script_data_as_string"`.

Reply:
Meh, I think it's not that important. If it's truthy, that's probably good enough for this test.

Reply:
This assertion was improved.